storybook/theme-switcher-addon/tokens.ts

```diff
@@ -0,0 +1,197 @@
+// @TODO - these should use tokens & values from @kaizen/design-tokens
+// when we are able to support multiple themes. For now, the Heart
+// theme tokens are hardcoded here
```

Comment:
Nice comment to clarify the cleanup required 👍

Reply:
I'm going to assume the values here are fine, given we'll switch to the real package later